modules/role/examples/cross_account_role.tf

```diff
@@ -20,7 +20,7 @@ resource "random_string" "external_id" {
 }
 
 module "sns" {
-  source     = "git@github.com:rackspace-infrastructure-automation/aws-terraform-sns//?ref=v0.0.1"
+  source     = "git@github.com:rackspace-infrastructure-automation/aws-terraform-sns//?ref=master"
```

Comment:
Looks like v0.0.1 is the current release.  I think we can probably just skip this repo.

Reply:
Ok.